frontend/src/router.js

```diff
@@ -264,13 +375,9 @@ export class Router {
                         const layoutHtml = await fetch(newRoute.useLayout).then(res => res.text());
```

Comment:
```suggestion
                        this.contentPageElement.innerHTML = await fetch(newRoute.useLayout).then(res => res.text());
```
переменную layoutHtml можно убрать вообще